tools/format.py

```diff
@@ -48,24 +48,16 @@ def main():
         did_fmt = True
 
     if not did_fmt:
-        prettier()
+        dprint()
         yapf()
         rustfmt()
 
 
-def prettier():
-    script = os.path.join(third_party_path, "node_modules", "prettier",
-                          "bin-prettier.js")
-    source_files = get_sources(root_path, ["*.js", "*.json", "*.ts", "*.md"])
-    if source_files:
-        max_command_length = 24000
-        while len(source_files) > 0:
-            command = ["node", script, "--write", "--loglevel=error", "--"]
-            while len(source_files) > 0:
-                command.append(source_files.pop())
-                if len(" ".join(command)) > max_command_length:
-                    break
-            run(command, shell=False, quiet=True)
+def dprint():
+    executable_path = get_prebuilt_tool_path("dprint")
+    chmod_executable(executable_path)
```

Comment:
This was necessary to get it running on the CI. I'll add a comment when responding to other feedback.